contracts/deposit-contract/depositContract.v.py

```diff
@@ -15,26 +15,17 @@
     index: bytes[8],
 })
 
-MIN_DEPOSIT_AMOUNT: public(uint256) # Gwei
-zero_hashes: bytes32[DEPOSIT_CONTRACT_TREE_DEPTH]
 branch: bytes32[DEPOSIT_CONTRACT_TREE_DEPTH]
-deposit_count: public(uint256)
+deposit_count: uint256
```

Comment:
Can this stay public? We read this value for the testnet website.

Reply:
Access to this appears to be through a function now:

```
@public
@constant
def get_deposit_count() -> bytes[8]:
      return self.to_little_endian_64(self.deposit_count)
```

Reply:
Thanks!